filament/backend/src/opengl/platforms/PlatformWGL.cpp

```diff
@@ -127,7 +121,22 @@ Driver* PlatformWGL::createDriver(void* const sharedGLContext,
 
     wglCreateContextAttribs =
             (PFNWGLCREATECONTEXTATTRIBSARBPROC) wglGetProcAddress("wglCreateContextAttribsARB");
-    mContext = wglCreateContextAttribs(whdc, (HGLRC) sharedGLContext, attribs);
+
+    int attribs[] = {
+            WGL_CONTEXT_MAJOR_VERSION_ARB, 4,
+            WGL_CONTEXT_MINOR_VERSION_ARB, 1,
+            0
+    };
+
+    // try all versions down, from GL 4.5 to 4.1
+    for (int minor = 5; minor >= 1; minor--) {
+        attribs[3] = minor;
```

Comment:
attribs[1] here?

Reply:
Oh yeah, good catch!

Reply:
uh, I'm confused why? `attribs[1]` is for `WGL_CONTEXT_MAJOR_VERSION_ARB` and contains `4`.

Reply:
haha my bad. Misread it.

Reply:
Oh yeah^2, you're right, I always forget the keys are in the array :)